metrics/deploy/rbac.yaml

```diff
@@ -73,6 +73,14 @@ rules:
     - get
     - list
     - watch
+- apiGroups:
```

Comment:
these RBAC changes should also be added to https://github.com/red-hat-storage/ocs-operator/blob/2f5040f0050e8a33c3ae4e6c44570133cfb8cdc6/config/rbac/exporter-role.yaml for it to be applied to the cluster.

Reply:
done.